datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -0,0 +1,278 @@
+package datadog
+
+import (
+	"errors"
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceDatadogSecurityMonitoringDefaultRule() *schema.Resource {
+	return &schema.Resource{
+		Exists: resourceDatadogSecurityMonitoringDefaultRuleExists,
+		Create: resourceDatadogSecurityMonitoringDefaultRuleCreate,
+		Read:   resourceDatadogSecurityMonitoringDefaultRuleRead,
+		Update: resourceDatadogSecurityMonitoringDefaultRuleUpdate,
+		Delete: resourceDatadogSecurityMonitoringDefaultRuleDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"rule_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the rule.",
+				ForceNew:    true,
+			},
+
+			"case": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "Cases of the rule, this is used to update notifications.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"status": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Status of the rule case to match.",
+						},
+						"notifications": {
+							Type:        schema.TypeList,
+							Required:    true,
+							Description: "Notification targets for each rule case.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
+
+			"enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Enable the rule.",
+			},
+
+			"disabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Disable the rule.",
```

Comment:
Same as the filter, would need some more details.

Reply:
I think you missed this `enabled/disabled` actually 🤔

Reply:
@Bit-Doctor fixed